### PR TITLE
replace Supervisor.Spec with child_spec for PG2

### PR DESCRIPTION
**Pull request by pragdave**
Addresses issue #94 

Not sure how to test it directly, but it seems to be indirectly tested thanks to the change in Phoenix.Supervise.

Dave

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,20 +16,23 @@ def application do
...
 ```elixir
 defmodule MyApp do
   use Application
 
   def start(_type, _args) do
-    import Supervisor.Spec, warn: false
 
     children = [
-      supervisor(Phoenix.PubSub.PG2, [MyApp.PubSub, []])
+      { Phoenix.PubSub.PG2, [ MyApp.PubSub, [] ]}
```
Comment (michalmuskala) [inline README.md]: I believe the preferred style would be without spaces around `{}[]`

**Comment by michalmuskala** [comment]
The project nominatively still supports elixir 1.2, so it shouldn't use the new child-specs internally without bouncing that requirement.

File: README.md (hunk 1 above)
Comment (pragdave) [reply README.md]: Might well be, but I don't use the formatter on religious grounds :)

Comment (josevalim) [reply README.md]: This project doesn't use the formatter. Even though, removing the spaces would make it consistent with the remaining code in the repository.

**Pull request closed without merging**